minio/console
# Fixed Bucket Events Notifications Page

fixes #2792 

## What does this do?

- Added missing ilm & replica supported types (According to MinIO Documentation, https://min.io/docs/minio/linux/reference/minio-mc/mc-event-add.html#id7)
- Fixed add screen crash when no arn is selected
- Migrated components to mds

## How does it look?

<img width="797" alt="Screenshot 2023-06-22 at 20 12 24" src="https://github.com/minio/console/assets/33497058/47d5da50-beb1-4c32-bbbe-f95d0490bc07">
<img width="1014" alt="Screenshot 2023-06-22 at 20 12 10" src="https://github.com/minio/console/assets/33497058/9d0e16f4-36a9-4cd7-9f49-6cac4b2be926">
<img width="1650" alt="Screenshot 2023-06-22 at 20 12 05" src="https://github.com/minio/console/assets/33497058/1a0c4c4c-3252-41fe-8768-62c23f53ebed">

## How to test?

1. Create a new Event destination (https://min.io/docs/minio/linux/administration/monitoring/bucket-notifications.html)
2. Create a Bucket and go into Events tab under Bucket Details page
3. Click on Subscribe to Event (You should see ILM & REPLICA options available)
4. Create the rule selecting the ARN created on step 1
5. Check the Events list, ilm & replica options should appear in the list if selected on step 3
